ckanext/scheming/tests/test_validation.py

```diff
@@ -219,15 +225,6 @@ def test_date_field_valid_date_datetime(self):
         )
         assert_equals(d['a_relevant_datetime'], '2014-01-01T12:35:00')
 
-    def test_date_field_valid_date_datetime(self):
-        lc = LocalCKAN()
-        d = lc.action.package_create(
-            type='camel-photos',
-            name='fred_datetime7',
-            a_relevant_datetime=datetime.datetime(2014, 1, 1, 12, 35),
-        )
-        assert_equals(d['a_relevant_datetime'], '2014-01-01T12:35:00')
-
```

Comment:
Why have you removed a test for the other datetime field? Why have you changed a number of tests above that test the datetime field?